tools/wptrunner/wptrunner/stability.py

```diff
@@ -292,14 +292,15 @@ def wrap_handler(x):
     # if the runs were stopped to avoid hitting the maximum run time.
     _, test_status = wptrunner.run_tests(**kwargs)
     iterations = test_status.repeated_runs
+    all_skipped = test_status.all_skipped
 
     logger._state.handlers = initial_handlers
     logger._state.running_tests = set()
     logger._state.suite_started = False
 
     log.seek(0)
     results, inconsistent, slow = process_results(log, iterations)
-    return results, inconsistent, slow, iterations
+    return results, inconsistent, slow, iterations, all_skipped
```

Comment:
I'm starting to think we should just return `test_status, results, inconsistent, slow` here and let the caller handle getting the data out of `test_status`.

Reply:
I went ahead and made [a small PR for this change](https://github.com/web-platform-tests/wpt/pull/32403). It could also be that just `log` and `test_status` can be returned from this function, and `process_results` is called in the `check_stability` loop. I am not certain what is best, but I will defer to the WPT gurus 🙇